ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -340,6 +344,13 @@ def configure_encryption(self):
             )
         self.do_click(self.dep_loc["next"], enable_screenshot=True)
 
+    def configure_data_protection(self):
+        """
+        Configure Data Protection
+
+        """
+        self.do_click(self.dep_loc["next"], enable_screenshot=True)
```

Comment:
Can we add a verifiication step to ensure it's actually enabled?

Reply:
@prsurve @am-agrawa Do you want to deploy RDR seup via UI? the RDR checkbox is unchecked by default...

Reply:
I am not sure. RDR deployment is a mix of UI and CLI. All I am saying is that if someone enables it via UI, we should have some checks to verify DR is enabled. Having it disabled by default is fine for normal deployments.